content/en/docs/concepts/services-networking/_index.md

```diff
@@ -23,9 +21,11 @@ implementation (barring any intentional network segmentation policies):
    * agents on a node (e.g. system daemons, kubelet) can communicate with all
      pods on that node
 
-Note: For those platforms that support `Pods` running in the host network (e.g.
+{{< note >}}
+For those platforms that support `Pods` running in the host network (e.g.
 Linux), when pods are attached to the host network of a node they can still communicate 
 with all pods on all nodes without NAT.
```

Comment:
Is this statement true? I believe it's wrong and the key ask was to correct that detail.

Reply:
done, could you please take a look.